README.md

```diff
@@ -1,24 +1,20 @@
-# Inter Protocol UI
+# dapp-psm
```

Comment:
this should still be "Inter Protocol UI"

Reply:
Fixed